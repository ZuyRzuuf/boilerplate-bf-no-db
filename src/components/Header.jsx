import React from 'react';

const Header = () => {
    return (
        <div>Backend/Frontend boilerplate without DB</div>
    );
};

export default Header;
