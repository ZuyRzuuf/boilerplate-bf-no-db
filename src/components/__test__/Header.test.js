import React from 'react';

import Header from '../Header';

describe('A suite', function() {
    test('renders isolated component correctly', () => {
        const wrapper = shallow(
            <Header />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders components tree correctly', () => {
        const wrapper = mount(
            <Header />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders HTML tree correctly', () => {
        const html = mount(
            <Header />
        );
        expect(html).toMatchSnapshot();
    });
});
