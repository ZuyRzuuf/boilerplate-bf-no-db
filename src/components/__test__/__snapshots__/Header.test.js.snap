// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite renders HTML tree correctly 1`] = `
<Header>
  <div>
    Backend/Frontend boilerplate without DB
  </div>
</Header>
`;

exports[`A suite renders components tree correctly 1`] = `
<Header>
  <div>
    Backend/Frontend boilerplate without DB
  </div>
</Header>
`;

exports[`A suite renders correctly 1`] = `
<div>
  Backend/Frontend boilerplate without DB
</div>
`;

exports[`A suite renders isolated component correctly 1`] = `
<div>
  Backend/Frontend boilerplate without DB
</div>
`;
